src/components/Ecosystem/Projects/index.tsx

```diff
@@ -0,0 +1,146 @@
+    <Grid item xs={12} md={4}>
+      <div className={css.card}>
+        <Avatar className={css.image} src={item.logo}>
+          &nbsp;
+        </Avatar>
+        <div className={css.socials}>
+          {item.twitter && (
+            <IconButton
+              aria-label="Twitter link"
+              href={item.twitter}
+              target="_blank"
+              rel="noreferrer"
+              size="small"
+              sx={{ color: 'primary.light' }}
+            >
+              <SvgIcon component={TwitterIcon} fontSize="inherit" color="inherit" inheritViewBox />
+            </IconButton>
+          )}
+          {item.github_dev_docs && (
+            <IconButton
+              aria-label="Github link"
+              href={item.github_dev_docs}
+              target="_blank"
+              rel="noreferrer"
+              size="small"
+              sx={{ color: 'primary.light' }}
+            >
+              <SvgIcon component={GithubIcon} fontSize="inherit" color="inherit" inheritViewBox />
+            </IconButton>
+          )}
+        </div>
+        <Typography fontWeight="500" mb={0.5}>
+          {item.project}
+        </Typography>
+        <Typography variant="body2" color="primary.light">
+          {item.value_prop}
+        </Typography>
+        <div className={css.categories}>
+          {categories &&
+            categories.map((category, idx) => {
+              return <Chip key={category + idx} className={css.categoryChip} label={category} />
+            })}
+        </div>
+      </div>
+    </Grid>
+  )
+}
+
+export const Projects = (): ReactElement => {
+  const projects = EcosystemDB as EcosystemProject[]
+
+  return (
+    <Container className={clsx(layoutCss.containerMedium, css.wrapper)}>
+      <Grid container>
+        <Grid item xs={12} md={8}>
+          <TextField
+            className={css.searchField}
+            variant="outlined"
+            placeholder="Search by name or category"
+            InputProps={{
+              startAdornment: (
+                <InputAdornment position="start">
+                  <SearchIcon />
+                </InputAdornment>
+              ),
+            }}
+            fullWidth
+          />
+          <Typography mt={2}>
+            <Typography component="span" color="primary.light">
+              Example:
+            </Typography>{' '}
+            DeFi, DAO Tooling, Payments, NFT, Infrastructure
+          </Typography>
+        </Grid>
+      </Grid>
+
+      <Divider sx={{ my: 9 }} />
+      <Grid container>
+        <Grid item xs={12} md={3}>
+          Sidebar
+        </Grid>
+        <Grid item xs={12} md={9}>
+          <Grid container spacing="30px">
+            {projects &&
+              projects.map((ecosystemProject) => {
+                return <Card key={ecosystemProject.project} {...ecosystemProject} />
+              })}
+          </Grid>
+        </Grid>
+      </Grid>
+      <Divider sx={{ mt: '100px', mb: '100px' }} />
```

Comment:
Let's make a note to check this fixed spacing on mobile.

Nit:

```suggestion
      <Divider sx={{ my: '100px' }} />
```